saleor/dashboard/product/forms.py

```diff
@@ -256,9 +272,17 @@ class Meta:
             'price_override': pgettext_lazy(
                 'Override price', 'Override price'),
             'name': pgettext_lazy('Product variant name', 'Name')}
+        help_texts = {
```

Comment:
Maybe `name` field shouldn't be included in the form, since it's not editable. It will be displayed somewhere in the page title or breadcrumbs anyway.